src/primitives/transaction.h

```diff
@@ -11,6 +11,8 @@
 #include "serialize.h"
 #include "uint256.h"
 
+static const unsigned int THEUNFORKENING = 257000;
```

Comment:
seems like an odd place for this kind of constant as it not really a transaction primitive but a consensus rule